templates/openharmony/entry/build-profile.json5

```diff
@@ -10,11 +10,12 @@
       ],
       "cppFlags": "",
     },
-    "compileMode": "esmodule",
+    "compileMode": "jsbundle",
   },
   "targets": [
     {
-      "name": "default"
+      "name": "default",
+      "runtimeOS": "HarmonyOS"
```

Comment:
Must use the harmonyos runtime environment.

Reply:
It seems wired.